src/index.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const state = {
+  temp: 75,
+  city: 'Seattle',
+  lat: '47.6038321',
+  long: '-122.330062',
+};
+
+// const axios = require('axios');
+// const cityForm = document.querySelector('form');
+// const landscape = document.querySelector('.card');
+// const card = document.querySelector('.details')
+
+const getLatAndLong = () => {
+  let lat, long;
+  axios.get('http://127.0.0.1:5000/location',
+  {
+      params: {
+        key: process.env['LOCATION_KEY'],
```

Comment:
I don't think you needed to pass in the `key` param for the call to the backend. That should have been on server side. In general, you never want to store valuable information like API keys in your frontend.